### DISCUSSION TITLE
Welcome to PumpKin Discussions!

### DISCUSSION THREAD

**Discussion by armarkos** [armarkos/PumpKin#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hyliyanling** [comment]
Compiling pumpkin-1.4 on windows using Cygwin gives this error "g++: error: unrecognized command-line option ‘-stdlib=libc++"
![1](https://github.com/armarkos/PumpKin/assets/163409988/eab291ae-744c-4fdc-91f4-5fd30e923f8b)

**Comment by dadogaspa** [comment]
Hello!

I have a problem making the package from source. When i call "make" it doesn't recognize the path in the makefile: /Users/aram/Documents/AHM_Projects/PumpKin/PumpKin/src/CMakeFiles/Makefile.cmake, what should I do? Maybe I should change the path with the one in my pc? but I don't have any CMakeFiles directory in my src, downloaded from GitHub (PumpKin-release-1.4)

I don't have much experience in programming, i need this code for my thesis project. If you could help me understand this I'd be very grateful

Thank you
Edoardo
